src/index.js

```diff
@@ -67,8 +67,8 @@ export Dropdown from './modules/Dropdown/Dropdown'
 import _Modal from './modules/Modal/Modal'
 export { _Modal as Modal }
 export const ModalContent = deprecateComponent('ModalContent', 'Use "Modal.Content" instead.', _Modal.Content)
-export const ModalFooter = deprecateComponent('ModalFooter', 'Use "Modal.Footer" instead.', _Modal.Footer)
 export const ModalHeader = deprecateComponent('ModalHeader', 'Use "Modal.Header" instead.', _Modal.Header)
+export const ModalActions = deprecateComponent('ModalActions', 'Use "Modal.Actions" instead.', _Modal.Actions)
```

Comment:
This deprecation message will show when `ModalActions` is imported.  Instead, we should export `ModalFooter` as deprecated:

```js
export const ModalFooter = deprecateComponent('ModalFooter', 'Use "Modal.Actions" instead.', _Modal.Actions)
```